apps/meteor/server/models/raw/Users.js

```diff
@@ -258,6 +258,112 @@ export class UsersRaw extends BaseRaw {
 		return this.findPaginated(query, options);
 	}
 
+	findPaginatedActiveUsersByRoomIdWithHighestRole(
+		searchTerm,
+		rid,
+		options,
+		searchFields,
+		extraQuery = [],
+		{ startsWith = false, endsWith = false } = {},
+	) {
+		if (options == null) {
+			options = {};
+		}
+
+		const termRegex = new RegExp((startsWith ? '^' : '') + escapeRegExp(searchTerm) + (endsWith ? '$' : ''), 'i');
+
+		const orStmt = (searchFields || []).reduce(function (acc, el) {
+			acc.push({ [el.trim()]: termRegex });
+			return acc;
+		}, []);
+
+		const query = {
+			$and: [
+				{
+					active: true,
+					__rooms: rid,
+					username: { $exists: true },
+					// if the search term is empty, don't need to have the $or statement (because it would be an empty regex)
+					...(searchTerm && orStmt.length > 0 && { $or: orStmt }),
```

Comment:
```suggestion
					...(searchTerm && orStmt.length && { $or: orStmt }),
```

More of a preference of playing with the boolean conversion of these

Reply:
Done! this is also the same we're doing on `findPaginatedByActiveUsersExcept`, so I changed it there too